### ISSUE TITLE
Moved my SYMPA to multi-domain mode and no more notification to listmaster when list creation

### ISSUE THREAD

**Issue by eurecomphil** [sympa-community/sympa#812]
Dear all,

I've switched SYMPA to multi-domain management.
Everything works perfectly except one point.
I am **listmaster** in **SYMPA.CONF** + **ROBOT.CONF** of all my domains.
When a standard user creates a list, I **do not receive anymore** an email for admin approval although it used to work when I was in a single domain management.

# LOGS
## SYMPA.LOG
When standard user creates list, it looks OK from SYMPA point of view :  
```
Nov 26 09:57:28 sympa wwsympa[11773]: [ID 702911 local1.info] info [robot lists.mydomain.fr] [session 38944343283271] [client 192.244.10.232] [user user.standard@mydomain.fr] main::do_create_list_request() do_create_list_request()
Nov 26 09:57:38 sympa wwsympa[11773]: [ID 702911 local1.info] info [robot lists.mydomain.fr] [session 38944343283271] [client 192.244.10.232] [user user.standard@mydomain.fr] main::do_create_list() do_create_list(pleaseignore2,pleaseignore2,confidential)
Nov 26 09:57:38 sympa wwsympa[11773]: [ID 702911 local1.info] info [robot lists.mydomain.fr] [session 38944343283271] [client 192.244.10.232] [user user.standard@mydomain.fr] main::do_create_list() do_create_list, get action : listmaster,notify 
Nov 26 09:57:38 sympa wwsympa[11773]: [ID 702911 local1.info] info admin::check_topics() admin::check_topics(computing,lists.mydomain.fr)
Nov 26 09:57:38 sympa wwsympa[11773]: [ID 702911 local1.info] info [robot lists.mydomain.fr] [session 38944343283271] [client 192.244.10.232] [user user.standard@mydomain.fr] [list pleaseignore2] main::do_create_list() notify listmaster
Nov 26 09:57:38 sympa wwsympa[11773]: [ID 702911 local1.info] info Auth::create_one_time_ticket() Auth::create_one_time_ticket(martin@mydomain.fr,lists.mydomain.fr,get_pending_lists,) value = 28340651379367
Nov 26 09:57:38 sympa wwsympa[11773]: [ID 702911 local1.info] info Auth::create_one_time_ticket() Auth::create_one_time_ticket(dupont@mydomain.fr,lists.mydomain.fr,get_pending_lists,) value = 18795413604513
```

## SMTP.LOG
I have absolutely nothing in SMTP.LOG according to this list creation action, although the fact that the mailing list is working fine and is able to send emails to members.

Any idea guys ? 
Thanks a lot in advance

**Comment by racke** [comment]
@eurecomphil we would be glad to hear why you closed the ticket.

**Comment by eurecomphil** [comment]
Dear Racke,

It is true that I could have been more explicit here (sorry I was in a hurry this morning and I did not wanted to let this ticket open for nothing).
I saw no errors at all from SYMPA nor POSTFIX. Everything looked working as usual but no emails were delivered by SYMPA. I was manually able to send emails by SENDMAIL, though.
A classical but efficient reboot of my Virtual Machine hosting SYMPA helped me to resolve this issue...  Once SYMPA has restarted all undelivered emails have been sent and everything is back to business again with no loss (apparently).

I just cross my fingers that it won't happen again soon.
Thank you Racke
